fetch.bs

```diff
@@ -6667,8 +6632,8 @@ enum ResponseType { "basic", "cors", "default", "error", "opaque", "opaqueredire
 <a for=Response>response</a>'s <a for=response>body</a>.
 
 <p>The
-<dfn constructor for=Response id=dom-response><code>Response(<var>body</var>, <var>init</var>)</code></dfn>
-constructor, when invoked, must run these steps:
+<dfn constructor for=Response id=dom-response lt="Response(body, init)"><code>new Response(<var>body</var>, <var>init</var>)</code></dfn>
+constructor steps are:
 
 <ol>
  <li><p>If <var>init</var>["{{ResponseInit/status}}"] is not in the range <code>200</code> to
```

Comment:
This step could be slightly modernized, by linking RangeError and de-codeifying the numbers.

Reply:
Hmm, currently all status codes are done like this, but I can see how we might not want to do that.